src/main/java/com/ecore/roles/web/rest/MembershipsRestController.java

```diff
@@ -30,25 +31,22 @@ public ResponseEntity<MembershipDto> assignRoleToMembership(
             @NotNull @Valid @RequestBody MembershipDto membershipDto) {
         Membership membership = membershipsService.assignRoleToMembership(membershipDto.toModel());
         return ResponseEntity
-                .status(200)
+                .status(201)
                 .body(fromModel(membership));
     }
 
     @Override
-    @PostMapping(
+    @GetMapping(
```

Comment:
In order to follow the Rest specification, of course, if the there isn’t any other security reason or URL length limitation